drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/BrowserCapabilitiesList.java

```diff
@@ -230,5 +222,27 @@ public int getPrecedence() {
         }
     }
 
-    ;
+    public static class chromeHeadless implements BrowserCapabilities {
+        @Override
+        public String getReadableName() {
+            return "chromeHeadless";
+        }
+
+        @Override
+        public String getImplementationClassName() {
+            return "org.openqa.selenium.chrome.ChromeDriver";
+        }
+
+        @Override
+        public Map<String, ?> getRawCapabilities() {
+            DesiredCapabilities headlessChrome = DesiredCapabilities.chrome();
+            headlessChrome.setBrowserName("chromeHeadless");
```

Comment:
Tried adding different properties and checking if chrome headless version supports reusing browser session but did not find any concrete reason for it.
There is no evidence to support that it is not does not support browser session reuse.